packages/esm-patient-medications-app/src/order-basket/medication-order-form.component.tsx

```diff
@@ -134,7 +134,6 @@ export default function MedicationOrderForm({
                       id: orderBasketItem.dosage.dosage,
                       text: orderBasketItem.dosage.dosage,
                     }}
-                    placeholder={t('editDoseComboBoxPlaceholder', 'Dose')}
```

Comment:
Why are these removed as part of this PR?

Reply:
placeholder is still valid in carbon. The typing library is currently incorrect. Instead for the time being we can use // @ts-ignore [like Vineet has done](https://github.com/openmrs/openmrs-esm-patient-chart/pull/675/files#diff-35064635b8bebb6e593e6d8dd084fee5577c4427ed31f2974f03632440c285b3)

Reply:
@ZacButko this is an outdated change... I believe you have already seen and responded to my message on Slack to Vineet? So basically I have locked the version to 7.49.0 with some coverage (no need to use @ts-ignore and be on infected version with missing coverage) - here is [my commit](https://github.com/openmrs/openmrs-esm-patient-chart/pull/667/commits/87f615553b401308150d86020270193ac42e74d4). I have also raised [this PR](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/60074) to fix `@types/carbon-component-react`

Reply:
@N0-man yes, sorry, awesome. I attempted to start reviewing your PR and then was taken away again. Thank you for both of those things